### DISCUSSION TITLE
Error, Error, Error

### DISCUSSION THREAD

**Discussion by maximilian22x** [mlrun/mlrun#7025] [Q&A] [answered]
I tried to install mlrun and import it. But it does not working. 

1. Why it only support Python 3.9 ? Why it is not up to date?
2. I install it on Python 3.9 but i can after that not import it: **AttributeError: partially initialized module 'mlrun' has no attribute 'common' (most likely due to a circular import)**

**Comment by maximilian22x** [comment, marked as answer]
I found the solution. Now it works: 

```
import os
os.environ["GIT_PYTHON_REFRESH"] = "quiet"
import git
import mlrun
```
https://stackoverflow.com/questions/48399498/git-executable-not-found-with-gitpython-bad-git-executable/55943328#55943328

**Comment by liranbg** [comment]
@maximilian22x Hi, thanks for updating with the hotfix.
About question 1 - We are focusing on preparing the ground work for python 3.11, I believe it would be available within the next minor version (1.9).
